src/components/schedule-visualizer/index.tsx

```diff
@@ -217,6 +229,50 @@ export default function ScheduleVisualizer(props: ScheduleVisualizerProps): Reac
   function handleBaseLayerChange(e: L.LayersControlEvent): void {
     debug('set current level name');
     setCurLevelName(e.name);
+    const mapWidth = document.getElementsByClassName('leaflet-image-layer')[0].clientWidth;
+    const mapHeight = document.getElementsByClassName('leaflet-image-layer')[0].clientHeight;
+    setMapDimensions({ width: mapWidth, height: mapHeight });
```

Comment:
I think this will return the viewport of the image layer, and not the map dimensions.